app/install.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -e
 
+# This assumes first time setup. Needs Optimizations.
```

Comment:
I was gonna mention that. When I try to run the script second time, it shows that 'directory already exists and not empty' . So, can we add a simple check to avoid it. If directory already present, do not clone it. Something like that.